pyramid_oereb/lib/renderer/extract/templates/xml/extract.xml

```diff
@@ -69,10 +69,10 @@
         <data:CantonalLogo>${extract.cantonal_logo.encode()}</data:CantonalLogo>
         <data:MunicipalityLogo>${extract.municipality_logo.encode()}</data:MunicipalityLogo>
     %else:
-        <data:LogoPLRCadastreRef>${request.route_url('{0}/image/logo'.format(route_prefix), logo='oereb')}</data:LogoPLRCadastreRef>
-        <data:FederalLogoRef>${request.route_url('{0}/image/logo'.format(route_prefix), logo='confederation')}</data:FederalLogoRef>
-        <data:CantonalLogoRef>${request.route_url('{0}/image/logo'.format(route_prefix), logo='canton')}</data:CantonalLogoRef>
-        <data:MunicipalityLogoRef>${request.route_url('{0}/image/municipality'.format(route_prefix), fosnr=extract.real_estate.fosnr)}</data:MunicipalityLogoRef>
+        <data:LogoPLRCadastreRef>${request.route_url('{0}/image/logo'.format(route_prefix), logo='oereb') | x}</data:LogoPLRCadastreRef>
+        <data:FederalLogoRef>${request.route_url('{0}/image/logo'.format(route_prefix), logo='confederation') | x}</data:FederalLogoRef>
+        <data:CantonalLogoRef>${request.route_url('{0}/image/logo'.format(route_prefix), logo='canton') | x}</data:CantonalLogoRef>
```

Comment:
Where does the value for `x` come from?

Reply:
Its a mako mechanism to handle strings which need to comply with the desired target format of output:
https://docs.makotemplates.org/en/latest/filtering.html

Reply:
Ah ok, thanks a lot!